templates/centos6.tpl.yaml

```diff
@@ -6,10 +6,8 @@ metadata:
   annotations:
     openshift.io/display-name: "CentOS 6.0+ VM"
     description: >-
-      This template can be used to create a VM suitable for
-      CentOS 6 and newer.
-      The template assumes that a PVC is available which is providing the
-      necessary CentOS disk image.
+      Template for CentOS 6 VM or newer.
+      A PVC with the CentOS disk image should be available.
```

Comment:
I believe "must be available" is the proper form here (the same in all the other templates).

Reply:
Changed